dataflow/gpu-workers/README.md

```diff
@@ -0,0 +1,168 @@
+# Workers with GPUs
```

Comment:
Is there a reason why this is a README and isn't a tutorial on devsite? Just want to make sure it's discoverable to all Dataflow users :)

Reply:
Hey, Leah! This will be hosted on devsite. Ours docs and samples deadlines have shifted out of sync, so I'll probably need to publish the docs on cloud.google.com before David can check in this sample.